src/neural/network_rr.cc

```diff
@@ -91,5 +93,197 @@ std::unique_ptr<Network> MakeRoundRobinNetwork(
 
 REGISTER_NETWORK("roundrobin", MakeRoundRobinNetwork, -999)
 
+class RecordComputation : public NetworkComputation {
```

Comment:
This should go to a separate file, say network_record.cc

Reply:
Done.